ballista/rust/client/src/context.rs

```diff
@@ -458,13 +469,16 @@ mod tests {
 
     #[tokio::test]
     #[cfg(feature = "standalone")]
+    #[ignore]
```

Comment:
This test is failing 🦜 😢 : https://github.com/apache/arrow-datafusion/issues/1840

Reply:
yeah, that's where I got stuck as well :). I can take a look at the failing test and fix it another PR. Thanks!